src/pages/Home/FeaturedPhotos.js

```diff
@@ -66,11 +66,14 @@ const FeaturedText = styled.p`
   font-size: 0.825rem;
   line-height: 1.36em;
   font-weight: 300;
+  @media ${device.mobile} {
```

Comment:
I'm assuming the styling changes here are due to needing to make this UI work in different breakpoints? if so I'd recommend using `rem` instead of `px`, instead.

Reply:
Also, 11px is a bit hard to read. Wondering if we could bump this to 12px at least?

Reply:
ooh, there was a reason for this too but I can't remember now. I believe there was some overlap onto the hero image. I wonder if adding more margin with larger text might be a better solution? I'll have to play around with this some more